testament/specs.nim

```diff
@@ -431,33 +430,32 @@ proc parseSpec*(filename: string,
           result.useValgrind = disabled
       of "disabled":
         case e.value.normalize
-        of "y", "yes", "true", "1", "on": result.err = reDisabled
+        of "y", "yes", "true", "1", "on":
+          result.disabledOs = {low(OsPlatform)..high(OsPlatform)}
+          result.disabledCpu = {low(CpuPlatform)..high(CpuPlatform)}
         of "n", "no", "false", "0", "off": discard
         of "win", "windows":
-          when defined(windows): result.err = reDisabled
+          result.disabledOs.incl OsPlatform.windows
         of "linux":
-          when defined(linux): result.err = reDisabled
+          result.disabledOs.incl OsPlatform.linux
         of "bsd":
-          when defined(bsd): result.err = reDisabled
+          result.disabledOs.incl {netbsd, freebsd, openbsd}
         of "osx", "macosx": # xxx remove `macosx` alias?
```

Comment:
On the `macosx` alias `xxx`: we can print warning/here for the time being

Reply:
I've searched the tests, and it seems like there's no usage of `macosx` in any `disabled` specification. I'm going to make a follow-up for removing the alias.